<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QWidget" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1508</width>
    <height>1009</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 252);

</string>
  </property>
  <widget class="QLabel" name="userL">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>300</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="passL">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>330</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="passRL">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>360</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Repeat Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="genreLab">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>410</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Select Your Preffered Genres:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="genreL">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>430</y>
     <width>461</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
selection-background-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);</string>
   </property>
   <property name="title">
    <string>Genre</string>
   </property>
   <widget class="QCheckBox" name="classics">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Classics</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="arabLit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Arabic Literature</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="philo">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Philosophy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="poetry">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Poetry</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="palestine">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Palestine</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="arabAmerican">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Arab-American</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="comic">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Comic Books</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="userLE">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>300</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="passLE">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>330</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="passRLE">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>360</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="regB">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>550</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backB">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>550</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="userError">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>300</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#941100;&quot;&gt;user already exists&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="passError">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>360</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#941100;&quot;&gt;password does not match&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="preferenceError">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>410</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#941100;&quot;&gt;* Please select at least one preference&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnB">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>640</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Return to Home Page</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginB">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>640</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Login as Registered Admin</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
